api/api/spec/spec.yaml

```diff
@@ -1519,7 +1934,7 @@ components:
         type: string
     format:
       in: query
-      name: fromat
+      name: format
       description: Optional. Output format (JSON, XML).
```

Comment:
Explain if this is a filter or give a bit more detail.

Reply:
Done in 12c4f03ef7687523273d303756ee5d983fc69bbb